script.js

```diff
@@ -0,0 +1,36 @@
+const firstName = document.getElementById("firstName");
+const lastName = document.getElementById("lastName");
+const age = document.getElementById("age");
+const email = document.getElementById("email");
+const phoneNum = document.getElementById("phoneNum");
+const male = document.getElementById("male");
+const female = document.getElementById("female");
+let errorMsg = document.getElementsByTagName("p")[0];
+let submit = document.getElementById("submit");
```

Comment:
@Arzoo-25  Please change the variable name `submit` refer above comment. Additionally, switch from using `let` to `const`. It's worth noting that the button returns an object with the `innerText` key, which we are modifying `const` will work here.